components/storage/ComparisonCards.vue

```diff
@@ -1,9 +1,41 @@
 <template>
-  <div>cards</div>
+  <div>
+    <template v-for="(service, j) in services">
+      <DCard
+        :key="'item' + j"
```

Comment:
I would use `service.service` as the key here (I think it's non-spaced) as these cards may change over time and if we use an index there's a risk that rendering could get weird as components get reused.